src/Sulu/Bundle/CoreBundle/Command/ServerCommandTrait.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CoreBundle\Command;
+
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+trait ServerCommandTrait
```

Comment:
why use a trait for that? you could easily put this into the command because i think this wont be reused in the near future